### PR TITLE
DOC-2270: add improvement documentation for TINY-10589 to the TinyMCE 7.0 release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2270

Site: [DOC-2270_TINY-10589 site](http://docs-feature-70-doc-2270tiny-10589.staging.tiny.cloud/docs/tinymce/latest/7.0-release-notes/#change-table-height-resizing-handling-to-remove-heights-from-tdth-elements-and-only-apply-to-tr-elements)

Changes:
* add improvement documentation for TINY-10589 to the TinyMCE 7.0 release notes.

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 1]
```diff
@@ -217,6 +217,12 @@ For more information about `highlight_on_focus` see the xref:accessibility.adoc#
 [NOTE]
 Any editors using this `highlight_on_focus: true` option, can remove this option from their {productname} init configuration when upgrading to {productname} 7.0.
 
+=== Change table height resizing handling to remove heights from `td`/`th` elements and only apply to `tr` elements.
+
+Previously, when resizing a table using either the row resize bars or corner resize handles, any existing `height` styles are removed from the `td`/`th` elements and only apply to `tr` elements and table element. Additionally, the heights calculated and applied for the `tr` elements and table element more accurately represent the actual visual height in the editor.
```
Comment (shanmen-tiny) [inline modules/ROOT/pages/7.0-release-notes.adoc:222]: It seems like the previous behaviour doesn't remove any of the existing `height` styles from the `td`/`th` elements. both the cells and the rows (`tr`) would retain their set heights.

Comment (ltrouton) [inline modules/ROOT/pages/7.0-release-notes.adoc:222]: I think we just need to remove the word 'Previously,' here
```suggestion
When resizing a table using either the row resize bars or corner resize handles, any existing `height` styles are removed from the `td`/`th` elements and only apply to `tr` elements and `table` element. Additionally, the heights calculated and applied for the `tr` elements and `table` element more accurately represent the actual visual height in the editor.
```

File: modules/ROOT/pages/7.0-release-notes.adoc [hunk 2]
```diff
@@ -217,6 +217,12 @@ For more information about `highlight_on_focus` see the xref:accessibility.adoc#
...
+
+As part of the changes to table height resizing handling, resizing via the corner resize handles is more consistent. If the table *does not* have any height styles and the table is resized using one of the corner resize handles, each row has its height increased evenly. If the table *does have* existing height styles, corner resizes will only change the height of the last row equivalent to `table_column_resizing: 'resizetable'` option for columns xref:table-options.adoc#table_column_resizing[Table options: table_column_resizing]
```
Comment (ltrouton) [inline modules/ROOT/pages/7.0-release-notes.adoc:224]: Sorry, I forgot to update the notes in the Jira
```suggestion
As part of the changes to table height resizing handling, resizing via the corner resize handles is more consistent. If the table *does not* have any height styles and the table is resized using one of the corner resize handles, each row has its height increased evenly. If the table *does have* existing height styles, corner resizes will only change the height of the first row (if using top corner handles) or last row  (if using bottom corner handles) equivalent to `table_column_resizing: 'resizetable'` option for columns xref:table-options.adoc#table_column_resizing[Table options: table_column_resizing]
```

**Review by ltrouton — APPROVED** [review]

File: modules/ROOT/pages/7.0-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.0-release-notes.adoc]: @ltrouton Can you update the above?

Comment (shanmen-tiny) [reply modules/ROOT/pages/7.0-release-notes.adoc]: It's all good now since @ltrouton has removed "previously", thanks!

**Review by shanmen-tiny — APPROVED** [review]

**Pull request merged**